### ISSUE TITLE
Keyboard navigation in results list, well, doesn't work

### ISSUE THREAD

**Issue by GabeDeBacker** [microsoft/sarif-vscode-extension#232]
While prototyping a fix for #231, which will leave "focus" in the explorer view, you quickly discover that while focus stays in the results list viewer, you cannot use the arrow keys to navigate between them.
